cumulus_library/cli.py

```diff
@@ -138,12 +140,26 @@ def create_template(path: str) -> None:
 
 
 def get_study_dict(alt_dir_paths: List) -> Optional[Dict[str, PosixPath]]:
-    """Convenience function for getting directories in ./studies/
+    """Gets valid study targets from ./studies/, and any pip installed studies
 
     :returns: A list of pathlib.PosixPath objects
     """
     manifest_studies = {}
-    paths = [Path(Path(__file__).resolve().parents[0], "studies")]
+    cli_path = Path(__file__).resolve().parents[0]
+
+    # first, we'll get any installed public studies
+    with open(Path(cli_path, "./module_allowlist.json"), "r") as study_allowlist_json:
+        study_allowlist = json.load(study_allowlist_json)
+    study_allowlist.pop("__desc__")
+    site_packages_dir = sysconfig.get_path("purelib")
```

Comment:
ok so - lemme get to the second part first. right now, for ease of end users, even though we're installing studies with pip, they aren't really pure python modules at the moment, since we're using the manifest.toml as a starting point to disambiguate out. I :could: add a wrapper at the base level to expose the toml infrastructure, but i kind of like keeping it out for two reasons:
- changes to the loader infrastructure only happen in the core repo
- one less bit of plumbing exposed to the 'analyst' user role

so, with that said: i'm targeting purelib based on the [list of python installation paths(https://docs.python.org/3/library/sysconfig.html#installation-paths). This path picks up the correct locationbased on whether i'm invoking with system python or a virtual env, so i believe it would work with package manager pythons as well.

since this path is down in the guts of the system, I'm somewhat loathe to expose it to the analyst class of users as an env var, though I would be ok doing so if we provided a command for them to run to get the contents of that var. 

if you buy this approach, i'll document this more in code, though please feel free to push back.

Reply:
> purelib

Sure, that works until someone has some *serious* study that ships some compiled code. Or we become popular and debian packages us, so the analyst apt-installs us, but pip-installs a specific study they want. Or the analyst installed the study into a venv that also sees their user-pip location. Just saying Python is specifically designed to avoid making assumptions about where imports live - the import flow chart is complicated.

You talk about not wanting wrappers - but isn't that what the `cumulus_library_covid` package is? Like it's a pip-installable wrapper package? I would have thought the right side of the dict could just be `cumulus_library_covid`, you import it, and then grab the toml dir -- either as a well-known folder under the wrapper package, or a defined variable in the wrapper or something, or a second component of the dictionary value.

> I'm somewhat loathe to expose it to the analyst class of users as an env var

Yeah totes agree. My mention of `PYTHONPATH` was just one example of the ways that python path importing is not a simple "one directory" thing.

Reply:
> I would have thought the right side of the dict could just be cumulus_library_covid, you import it, and then grab the toml dir -- either as a well-known folder under the wrapper package, or a defined variable in the wrapper or something, or a second component of the dictionary value.

honestly, i hadn't thought of this approach - lemme give this a go.

Reply:
although - this does create a nested 'import this thing if it exists' headache - right now these things are agnostic to importing.

Reply:
> right now these things are agnostic to importing.

Says the guy inspecting `purelib` :smile:

Reply:
Sure yeah